themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md

```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
 `-y`, `--yes`
 
 :   Use to specify if the Chef Infra Server can go offline during
-    tar.gz-based backups.
\ No newline at end of file
+    tar.gz-based backups.
+
+`--pg-options`
+
+:   Specify additional options to pass to PostgreSQL during backups.
```

Comment:
could we include some examples of the options?

Reply:
We will skip this for now, as discussed in standup.